### ISSUE TITLE
Windows: Windows key should not be displayed as "Meta"

### ISSUE THREAD

**Issue by sgarcialaguna-mms** [zealdocs/zeal#594]
Under Options->General->Global shortcuts->Show Zeal, the default setting under Windows is "Meta + Z". 

As a Windows user, this is confusing, eventually I found out that Meta refers to the Windows key.

**Comment by trollixx** [comment]
You are right! The reason is a very old [QTBUG-4181](https://bugreports.qt.io/browse/QTBUG-4181).
